semgrep/semgrep/target_manager_extensions.py

```diff
@@ -33,6 +33,7 @@
 CSHARP_EXTENSIONS = [FileExtension(".cs")]
 RUST_EXTENSIONS = [FileExtension(".rs")]
 KOTLIN_EXTENSIONS = [FileExtension(".kt")]
+YAML_EXTENSIONS = [FileExtension(".yaml")]
```

Comment:
The the comment above about test_rule_parser.py --snapshot-update
This is why the CI is failing.